tests/functional/collection/role/test_role_delete.py

```diff
@@ -30,7 +30,7 @@ def test_successful_gcs_collection_role_delete(
 
     # now test the command and confirm that a successful role deletion is reported
     run_line(
-        ["globus", "gcs", "collection", "role", "delete", collection_id, role_id],
+        ["globus", "collection", "role", "delete", collection_id, role_id],
```

Comment:
Could your update the invocation prefix here & in the other test files to `["globus", "gcs", "collection", ...]

There's a bit of contexted, but those are namespaced intentionally.

---

Our ultimate vision for the cli is to move specific resource and action keys (i.e., "collection") out of the top-level namespace by always nesting them under their service name. That itself will definitely be a breaking change however, so initially we started by introducing `gcs` as an equivalent command to the existing ones ([in this PR](https://github.com/globus/globus-cli/pull/902)).

Given that direction, `globus gcs collection ...` is actually our primary invocation path, not `globus collection ...` so we should keep our tests using that.